app/src/main/res/values/strings.xml

```diff
@@ -1121,7 +1128,19 @@
     <string name="advanced_mode">Advanced Mode</string>
     <string name="reminder_title">Reminder Title</string>
     <string name="what_time_of_day_question">What time of day?</string>
+
+    <string name="all_alerts_currently_disabled">ALL ALERTS CURRENTLY DISABLED</string>
+    <string name="low_and_high_alerts_currently_disabled">LOW AND HIGH ALERTS CURRENTLY DISABLED</string>
+    <string name="low_alerts_currently_disabled">LOW ALERTS CURRENTLY DISABLED</string>
+    <string name="high_alerts_currently_disabled">HIGH ALERTS CURRENTLY DISABLED</string>
+    <string name="bg_reading_missed">BG Readings Missed</string>
+    <string name="not_connected">Not Connected</string>
+    <string name="no_data">No data</string>
+    <string name="no_bluetooth">No bluetooth</string>
+    <string name="connection_status">Connection Status:   </string>
```

Comment:
@tzachi-dar could you explain why you add three spaces at the end of the string? Normally something like that should be handled in the code? I'm just wondering, as this now has to take care in every translation.

Reply:
I did not change anything here, it existed before.
I only moved this to the resource file.
See https://github.com/NightscoutFoundation/xDrip/pull/679/files#diff-c8160ebde93fb72b4dec407c2c46b779L115

Reply:
Ah, OK. Just wondering, not a big thing...